Makefile

```
@@ -79,6 +88,21 @@ validate: $(BUF)
 	go build -o bin/protoc-gen-go-cybozu-validate ./cmd/protoc-gen-go-cybozu-validate
 	$(RUN_BUF) generate --template buf.go-cybozu-validate.gen.yaml
 
+.PHONY: es
+es: $(BUF) $(PROTOC_GEN_ES)
+	cd $(ES_PACKAGES) && npm ci
+	$(RUN_BUF) generate --template ./buf.es.gen.yaml
+	cd $(ES_PACKAGES_PROTOBUF) && npm run build
+
+.PHONY: es-validate
+es-validate: es
```

nit. I prefer calling `$(MAKE)` to run a target if the target is phony
as calling `$(MAKE)` is more consistent with other parts (see `all` for example).
```suggestion
es-validate:
	$(MAKE) es
```